docker/Dockerfile.proplet

```
@@ -0,0 +1,71 @@
+RUN mkdir -p /home/proplet && \
+    chown proplet:proplet /home/proplet
+
+# Switch to non-root user
+USER proplet
+
+# Set working directory for config file
+WORKDIR /home/proplet
+
+# Health check
+HEALTHCHECK --interval=30s --timeout=3s --start-period=5s --retries=3 \
+    CMD ["/usr/local/bin/proplet", "--help"] || exit 1
```

Hadolint (2.14.0)

[error] unexpected '|'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)